api/v1/models/User.js

```diff
@@ -47,13 +85,59 @@ User.prototype.setPassword = function (password) {
 };
 
 /**
+ * Retrieves a role from the roles field.
+ * @param  {String}	role the string representation of a role from utils.roles
+ * @return {UserRole}	 the desired role, or undefined
+ */
+User.prototype.getRole = function (role) {
+	return _.find(this.related('roles').models, function (role) {
+		return role.role === role;
+	});
+};
+
+/**
+ * Determines whether or not this user has the specified role
+ * @param  {String}	role the string representation of a role from utils.roles
+ * @param  {Boolean} activeOnly whether or not the role must be active (default true)
+ * @return {Boolean}	  whether or not the user has the role
+ * @throws {TypeError}	  when the user is missing its related roles key
+ */
+User.prototype.hasRole = function (role, activeOnly) {
+	if (_.isUndefined(this.related('roles'))) {
+		throw new TypeError("The related roles were not fetched with this User");
+	}
+
+	var roleMatch = { role: role };
+	if (_.isUndefined(activeOnly) || activeOnly) {
```

Comment:
`activeOnly` is optional, with a default value of `true`. So that snippet is checking to see if the caller passed anything (if not, the expression evaluates to `true` since `activeOnly` is `undefined`)